pkgs/by-name/re/rescrobbled/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, fetchFromGitHub, rustPlatform, pkg-config, openssl, dbus }:
+
+rustPlatform.buildRustPackage rec {
+
+  pname = "rescrobbled";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "InputUsername";
+    repo = "rescrobbled";
+    rev = "v${version}";
+    hash = "sha256-1E+SeKjHCah+IFn2QLAyyv7jgEcZ1gtkh8iHgiVBuz4=";
+  };
+
+  cargoLock.lockFile = src + "/Cargo.lock";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl dbus ];
+
+  checkFlags = [
+    "--skip=filter::tests::test_filter_script"
```

Comment:
For the script you can just patch the path: https://github.com/NixOS/nixpkgs/pull/153392/files#diff-5b60f7aa0239041e4bdb4c6be0ba2a3fa11e5ce0ebd67e7cf31e2c9d258713e3R32